workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/WorkflowInterceptor.kt

```diff
@@ -87,6 +87,16 @@ public interface WorkflowInterceptor {
     session: WorkflowSession
   ): S = proceed(old, new, state)
 
+  /**
+   * Intercept calls to [com.squareup.workflow1.internal.WorkflowRunner.nextRendering].
```

Comment:
```suggestion
   * Intercept calls to [WorkflowRunner.nextRendering][com.squareup.workflow1.internal.WorkflowRunner.nextRendering].
```

Reply:
This definitely needs kdoc here as we are intercepting an internal method in the Runner.

Reply:
done